src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -385,6 +385,7 @@ void Convolution::getSupportedDescriptors() {
             std::for_each(implPriorities.begin(), implPriorities.end(), [&](const impl_desc_type& desc_type) {
```

Comment:
I think `for_each` is not best option here.
It looks more like `find` or `any_of`, there is no need to iterate over all element just to first match condition then set flags

e.g.

```cpp
if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core) &&
    std::any_of(implPriorities.begin(), implPriorities.end(), [](const impl_desc_type& desc_type) {
        return static_cast<bool>(desc_type & impl_desc_type::brgconv_avx512);
     })) {
    shouldTryBrgconv = true;
    userForceBrgconv = true;
}
```

or I missed something?

Reply:
Good suggestion, updated.